cmd/scollector/collectors/fastly.go

```diff
@@ -0,0 +1,374 @@
+	Imgopto            int64   `json:"imgopto"`
+	Ipv6               int64   `json:"ipv6"`
+	Log                int64   `json:"log"`
+	MissTime           float64 `json:"miss_time" div:"true" rate:"gauge" unit:"seconds per second" desc:"The amount of time spent processing cache misses"`
+	OrigReqBodySize    int64   `json:"orig_req_body_size"`
+	OrigReqHeaderSize  int64   `json:"orig_req_header_size"`
+	OrigRespBodySize   int64   `json:"orig_resp_body_size"`
+	OrigRespHeaderSize int64   `json:"orig_resp_header_size"`
+	Otfp               int64   `json:"otfp"`
+	Pass               int64   `json:"pass" div:"true" rate:"gauge" unit:"hits per second" desc:"The number of requests that passed through the CDN without being cached"`
+	Pci                int64   `json:"pci"`
+	Requests           int64   `json:"requests" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of requests processed"`
+	Shield             int64   `json:"shield"`
+	StartTime          int64   `json:"start_time" exclude:""`
+	Status1xx          int64   `json:"status_1xx" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 1xx response code (Informational)."`
+	Status200          int64   `json:"status_200" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 200 response code (Success)."`
+	Status204          int64   `json:"status_204" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 204 response code (No Content)."`
+	Status2xx          int64   `json:"status_2xx" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 2xx response code (Success)."`
+	Status301          int64   `json:"status_301" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 301 response code (Moved Permanently)."`
+	Status302          int64   `json:"status_302" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 302 response code (Found)."`
+	Status304          int64   `json:"status_304" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 304 response code."`
+	Status3xx          int64   `json:"status_3xx" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 3xx response code (Redirection)."`
+	Status4xx          int64   `json:"status_4xx" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 4xx response code (Client Error)."`
+	Status503          int64   `json:"status_503" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a 503 response code. (Service Unavailable)"`
+	Status5xx          int64   `json:"status_5xx" div:"true" rate:"gauge" unit:"requests per second" desc:"The number of http requests with a xxx response code."`
+	Synth              int64   `json:"synth"`
```

Comment:
Would add description: 'Synthetic response sent from VCL. Typically used to send edge-generated error pages.'